readthedocs/settings/base.py

```diff
@@ -485,6 +485,7 @@ def USE_PROMOS(self):  # noqa
     # RTD Settings
     REPO_LOCK_SECONDS = 30
     ALLOW_PRIVATE_REPOS = False
+    RTD_ALLOW_PRIVATE_PROJECTS = False
```

Comment:
Seems `ALLOW_PRIVATE_REPOS` or `DEFAULT_PRIVACY_LEVEL` is what we want to use, instead of adding a new setting?

Reply:
Wanted to use a more general name and deprecate the other one, but I can change it to use `ALLOW_PRIVATE_REPOS`

Reply:
Yea, lets keep that for now, so we don't have _more_ settings, but we can rename it in the future if needed.